php/coresky/main/gate.php

```diff
@@ -0,0 +1,146 @@
+<?php
```

Comment:
Why not having a bootstrap command that generates this file ? Instead of having one to upload onto the container?

Reply:
Both of these files are generated **only at the development stage**. The development tool with web-interface is used. The information obtained using the web interface is stored in the PHP array and is part of the application code. Classic routing has been replaced in my framework with SkyGate. Try using a translator:
https://github.com/energy-coresky/air/wiki/%D0%9A%D0%BE%D0%BD%D1%82%D1%80%D0%BE%D0%BB%D0%BB%D0%B5%D1%80%D1%8B-%D0%B8-SkyGate